.azure-pipelines.yml

```diff
@@ -16,8 +16,12 @@
 #    https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/v2-windows
 
 trigger:
-- epochs/*
-- triggers/*
+- epochs/daily
```

Comment:
Curious why only these epochs - worth a comment maybe?

Reply:
They're the only ones referenced in the following config, I'll add a comment.